### PR TITLE
Add UG for Invalid Date warning, Add UG for creation of Meeting for P…

### PR DESCRIPTION
**Pull request by Brendan8899**
Invalid Date warning like 31-02-2023 for All Possible Commands, used existing commands
Add UG for creation of Meeting in previous Date
Add UG for creation of people of Duplicate Name, Email and Phone

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddScheduleCommandParser.java [hunk 1]
```diff
@@ -67,7 +66,7 @@ public AddScheduleCommand parse(String args) throws ParseException {
             try {
                 date = LocalDate.parse(
                         argMultimap.getValue(PREFIX_DATE).get(),
-                        DateTimeFormatter.ofPattern("dd-MM-uuuu").withResolverStyle(ResolverStyle.STRICT)
+                        DateTimeFormatter.ofPattern("dd-MM-yyyy")
```
Comment (tata32000) [inline src/main/java/seedu/address/logic/parser/AddScheduleCommandParser.java:69]: What is this for?

Comment (Brendan8899) [reply src/main/java/seedu/address/logic/parser/AddScheduleCommandParser.java:69]: I previously changed it to uuuu now I change it back to yyyy

Comment (Brendan8899) [reply src/main/java/seedu/address/logic/parser/AddScheduleCommandParser.java:69]: uuuu with resolver style is the correct way to do Leap year Date

**Comment by tata32000** [comment]
The DG seems to fail checkstyle

**Review by jewiteo — COMMENTED** [review]
Small comment for line 227 in the UG, else LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -218,10 +224,14 @@ Example:
 
 ![add schedule example](images/add-schedule_example.png)
 
-- `add-schedule c/1 2 n/Dinner d/10-10-2024 t/1800`
+- `add-schedule c/1 2 n/Dinner d/10-10-2024 t/1800`add
```
[the remark this answers is not in the document]
Comment (Brendan8899) [reply docs/UserGuide.md:227]: My bad thanks I will edit

**Review by tata32000 — APPROVED** [review]
LGTM

**Pull request merged**